src/popup/components/tabs/Tabs.vue

```diff
@@ -1,6 +1,9 @@
 <template>
-  <div class="tabs">
-    <slot />
+  <div>
+    <div class="tabs">
+      <slot />
+    </div>
+    <slot name="subTabs" />
```

Comment:
In my opinion this makes no sense to have "subTabs" slot in the Tabs component. It gives no value.